### PR TITLE
Navigation component cu 862jmeg08

### PR DESCRIPTION
**Pull request by OleksandrMykhailovskyi**
Here is the functional added to the PR: 

- Created mobile-nav component that is used in header.
- There are the animations and open/close effects there. Also, if the screen resolution changes from desktop to mobile and back, the elements are automatically closed.
- There is the typization for navigationData data.
- Created a custom Li component with CVA. It is used in mobile-nav component, inside our NavigationLink. This the way, how the HTML tags take place on vercel.com.
- Also, the real data is added to the mobile menu, except the buttons group, that does not have back realization yet.
- Checked all the imports for them to be generic ones.
- There are the styles for both light and dark themes.

### CODE REVIEW

**Comment by andrewmarushko** [comment]
Task linked: [CU-862jm1wvp Header](https://app.clickup.com/t/862jm1wvp)

**Comment by andrewmarushko** [comment]
Task linked: [CU-862jmeg08 Navigation Component](https://app.clickup.com/t/862jmeg08)

File: components/ui/li.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+const liVariants = cva(
+  "select-none flex items-center text-base h-geist-gap-x2 transition-background-color duration-200 ease-in-out dark:hover:bg-accent-100 dark:focus:bg-accent-100 dark:focus-visible:border-accent-900",
+  {
+    variants: {
+      variant: {
+        firstLevelLink: 
+          "text-accent-300 dark:text-accent-600 border-b-accent-800 dark:border-b-accent-200 border-b", 
+        secondLevelLink: 
+          'text-accent-400 dark:text-accent-500',
```
Comment (andrewmarushko) [inline components/ui/li.tsx:16]: используй menu and submenu нейминг вроде как болеее логично

**Review by andrewmarushko — CHANGES_REQUESTED** [review]

File: components/mobile-nav.tsx [hunk 2]
```diff
@@ -1,42 +1,83 @@
...
+            href={'/feedback'} 
+            onClick={hideMobileNav}
+          >
+            Feedback
+          </NavigationLink>
+        </li>
+      </ul>
+      <ul>
+        <div className='text-left border-b border-b-accent-800 dark:border-b-accent-200'>
+          <h3 className='font-medium text-base -tracking-widest leading-6'>
```
[the remark this answers is not in the document]
Comment (OleksandrMykhailovskyi) [reply components/mobile-nav.tsx:49]: на верселе как раз так было, брал оттуда структуру HTML

image

[the remark this answers is not in the document]
Comment (OleksandrMykhailovskyi) [reply components/mobile-nav.tsx:49]: заюзал Аккордеон

**Review by andrewmarushko — APPROVED** [review]

**Pull request merged**